packages/valory/agents/abstract_abci/aea-config.yaml

```diff
@@ -56,3 +57,19 @@ config:
 public_id: valory/abstract_abci:0.1.0
 type: skill
 is_abstract: false
+---
+public_id: valory/p2p_libp2p_client:0.1.0
+type: connection
+config:
+  nodes:
+  - uri: ${P2P_URI:str:staging.acn.autonolas.tech:9005}
+    public_key: ${P2P_PUBLIC_KEY:str:0250827315351f6f4d3777b77d4bfd67280156240313bf221750b36404344bb35d}
+cert_requests:
+- identifier: acn
+  ledger_id: ethereum
+  message_format: '{public_key}'
+  not_after: '2022-01-01'
+  not_before: '2021-01-01'
+  public_key: ${P2P_PUBLIC_KEY:str:0250827315351f6f4d3777b77d4bfd67280156240313bf221750b36404344bb35d}
+  save_path: .certs/acn_cosmos_9005.txt
+is_abstract: true
```

Comment:
why is this even here?

Reply:
https://github.com/valory-xyz/open-autonomy/pull/909#discussion_r899044567